components/list/list-item-drag-drop-mixin.js

```diff
@@ -252,6 +254,7 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 			<d2l-list-item-drag-handle
 				id="${this._itemDragId}"
 				class="${classMap(classes)}"
+				text="${this.dragHandleAriaLabel || this.localize('components.list.list-item-drag-handle')}"
```

Comment:
I would expect that the person using the list item would localize this. Does this make it easier?

Reply:
I figured having a default label was good because in most cases the label should be the same?

Reply:
I decided you were right

Reply:
Am I? My original thought was to have a default too. Didn't read this right. Blaming head ache.

Reply:
The static text, "Drag Handle" makes sense to me, and I'm not sure it needs to be customizable.  I wouldn't add to the API until we have a really compelling use-case (do we already?).  I think the "button" verbiage can be omitted, because it should already be announced as the element getting focus is an actual `button`.

If we did need to expose this, I would omit "Aria" from the property name and just expose it similar the the handle component's property - "DragHandleText", since Aria is an internal detail of `d2l-list-item-drag-handle`.

Reply:
We have a use case. We will want to give the position which is something unknown. Unless there is a better way to do it.

Reply:
This is what we have in the design: 
![image](https://user-images.githubusercontent.com/13232226/87587637-5ae93c80-c6b0-11ea-8815-f736883af05d.png)

Reply:
Another change, we will have to make is that the label for the drag handle. 
While it is in non-keyboard mode be the static default value.
When it switches to keyboard mode then we can add the custom so we have the position value.

Does that make sense?

Reply:
I am also happy if they are both static but if they are going to be static maybe just do it in the drag handle?

Reply:
I see.  Yeah so that has Activity specific verbiage.  Is "Activity current position 1/4" meaningful?  I guess that's a design question.  I'm not sure what "1/4" means... i.e. does it mean position 1 of 4?

Aside from that, I think we should be careful about baking the up/down verbiage into the app-specific terms.  I think that verbiage is best owed by the core component because that's where the interaction is implemented/defined.  I'm all for conveying that info, but if ever change the keyboard interaction we don't want to be having to fix lang-terms in apps.

Reply:
I agree with you

Reply:
I've renamed the attribute, and also changed the default text to `Reorder action` and `Reorder item`